pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools >= 66.1.0, <= 80.3.1", "wheel == 0.45.1"]  # lower bound to support controller Python versions, upper bound for latest version tested at release
+requires = ["setuptools >= 66.1.0, <= 80.3.1"]  # lower bound to support controller Python versions, upper bound for latest version tested at release
```

Comment:
If we're going to drop the wheel pin, we'll need to raise our minimum setuptools version to 70.1.0 to ensure the wheel package isn't needed.

Reply:
Given that setuptools 70.1.0 was released over a year ago, I don't think there should be any issues using that as the new minimum.

Worst case, if a package maintainer does need to use an older setuptools, they'll have to deal with any potential incompatibilities themselves -- which could happen since we'll no longer be testing our builds against a lower version.